### PR TITLE
deprecate old keyvault previews

### PR DESCRIPTION
**Pull request by mario-guerra**

### CODE REVIEW

**Comment by mario-guerra** [comment]
Hi @weshaggard, I don't know why/how the link identified as broken is being formed from the commit I submitted. Can you help identify what went wrong?

File: _data/releases/latest/java-packages.csv [hunk 1]
```diff
@@ -702,15 +702,15 @@
...
-"azure-keyvault-keys","com.azure","","4.0.0-preview.5","Key Vault - Keys","Key Vault","keyvault","NA","NA","client","false","","","","","","","com.azure/azure-security-keyvault-keys","","","",""
+"azure-keyvault-keys","com.azure","","4.0.0-preview.5","Key Vault - Keys","Key Vault","keyvault","NA","NA","client","false","","","","deprecated","3/31/2023","false","com.azure/azure-security-keyvault-keys","NA","","",""
 "azure-keyvault-parent","com.azure","","4.0.0-preview.2","Key Vault - Parent","Other","NA","NA","NA","client","false","","","","","","true","","","","","Never shipped GA"
 "azure-keyvault-parent","com.microsoft.azure","1.2.1","","Key Vault - Parent","Other","NA","NA","NA","client","false","","","","maintenance","","true","","","","",""
-"azure-keyvault-secrets","com.azure","","4.0.0-preview.5","Key Vault - Secrets","Key Vault","keyvault","NA","NA","client","false","","","","","","","com.azure/azure-security-keyvault-secrets","","","",""
+"","com.azure","","4.0.0-preview.5","Key Vault - Secrets","Key Vault","keyvault","NA","NA","client","false","","","","deprecated","3/31/2023","false","com.azure/azure-security-keyvault-secrets","NA","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:713]:
```suggestion
"azure-keyvault-secrets","com.azure","","4.0.0-preview.5","Key Vault - Secrets","Key Vault","keyvault","NA","NA","client","false","","","","deprecated","3/31/2023","false","com.azure/azure-security-keyvault-secrets","NA","","",""
```

Comment (weshaggard) [reply _data/releases/latest/java-packages.csv:713]: Looks like you accidently deleted the package name in this entry

Comment (mario-guerra) [reply _data/releases/latest/java-packages.csv:713]: Oops! Good catch Wes, thanks.

**Review by weshaggard — APPROVED** [review]

**Comment by mario-guerra** [comment]
Thanks for the assist @weshaggard !

**Pull request merged**